dts/arm/nxp/nxp_ke17z.dtsi

```diff
@@ -42,7 +42,7 @@
 			flash0: flash@0 {
 				compatible = "soc-nv-flash";
 				reg = <0 DT_SIZE_K(256)>;
-				erase-block-size = <DT_SIZE_K(2)>;
+				erase-block-size = <DT_SIZE_K(4)>;
```

Comment:
Why is this changed?

Reply:
The littlefs requires a block size of 4K, and 2K is not enough.

Reply:
This shouldn't be the case. Littlefs should work with a block size of 2kB. Probably there is some error in a littlefs test.

Reply:
I see that all the boards that support littlefs use erase-block-size of 4096, by changing the value of this attribute found that the value of stat.f_frsize in the test code [tests/subsys/fs/littlefs/src/test_lfs_basic.c](https://github.com/zephyrproject-rtos/zephyr/blob/d6afb8fb521844ff8a24d70f1432a428b975a1bd/tests/subsys/fs/littlefs/src/test_lfs_basic.c#L62) is from here, so it should be 4096. Increasing this value has no other effect.

Reply:
> I see that all the boards that support littlefs use erase-block-size of 4096

Do you mean that the value was set according to the littlefs SW, not a real HW?

Reply:
So if I want to test the littlefs, this change should be put in the tests/subsys/fs/littlefs boards overlay, right?

Reply:
You could try to do it in an overlay, but really the test is wrong.

Reply:
> So if I want to test the littlefs, this change should be put in the tests/subsys/fs/littlefs boards overlay, right?

Better to fix the littlefs limitation.

Reply:
Updated, fixed values replaced with macros, from the dts configuration. Please help to review, thanks!